common/changes/@microsoft/api-documenter/fix-newline-in-table-code-block_2019-04-15-10-59.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-documenter",
+      "comment": "Fixes newline rendering when code blocks are used in tables",
```

Comment:
Change entries should use imperative phrasing ("Fix newline rendering" instead of "Fixes newline rendering").  [This doc](https://rushjs.io/pages/best_practices/change_logs/) has more info.

Reply:
:+1: